### PR TITLE
Minimize confusion for Zyte API proxy mode users

### PR DESCRIPTION
**Pull request by Gallaecio**

### CODE REVIEW

File: scrapy_zyte_smartproxy/middleware.py [hunk 1]
```diff
@@ -220,6 +220,10 @@ def _translate_headers(self, request, targets_zyte_api):
                 request,
             )
 
+    def _inc_stat(self, stat, value=1, /, *, targets_zyte_api):
```
Comment (wRAR) [inline scrapy_zyte_smartproxy/middleware.py:223]: This breaks on 3.7 apparently

Comment (Gallaecio) [reply scrapy_zyte_smartproxy/middleware.py:223]: Yes. I did it knowingly, thinking this repo had dropped 3.7- support already :facepalm:

Comment (kmike) [reply scrapy_zyte_smartproxy/middleware.py:223]: Hey! It seems the main change in this PR is actually dropping old Python support :) How hard would it be to keep it, and do separately?

Comment (kmike) [reply scrapy_zyte_smartproxy/middleware.py:223]: I think we wanted to be pretty conservative with this package.

**Review by kmike — APPROVED** [review]

**Comment by Gallaecio** [comment]
OK, I better run the tests locally in a Python 2 Docker image and stop wasting CI time.

**Pull request merged**